windows/clangw/CMakeLists.txt

```diff
@@ -2,3 +2,4 @@
 # Licensed under the MIT License.
```

Comment:
Can we avoid introducing more top level folders? Consider `tools/winbuild` (clangw is also a little confusing since we also build llvm-arw as a separate tool from the same folder)

Also, I think we may have discussed this before, but why make these wrappers executables as opposed to just scripts that live under the `scripts` folder?

Reply:
Currently the tools folder consists of tools that are actually installed (oesign, oesgx, oeedger8r).
It is preferable to keep it that way.

I like the idea of using scripts for clang and LLVM wrappers. 
We don't yet have a good scripting language on Windows. powershell scripts won't work out of the box; I am not too familiar with cmd scripts. Maybe python is the way to go?

In any case, we can revisit it in future iterations as we use the Windows workflow more.

Reply:
Python on Windows also sucks 😄